io-native/src/main/java/org/ballerinalang/stdlib/io/nativeimpl/ByteChannelUtils.java

```diff
@@ -36,8 +36,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.ByteArrayInputStream;
-import java.io.IOException;
+import java.io.*;
```

Comment:
Do not include * imports. Seems checkstyle check is not run in the module. Add that too.

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/31/commits/f8d1a96efa9f17ac54b1fd2a145c10de275d9e94